SimCalorimetry/HcalTrigPrimAlgos/src/HcalTriggerPrimitiveAlgo.cc

```diff
@@ -553,6 +552,24 @@ void HcalTriggerPrimitiveAlgo::analyzeQIE11(IntegerCaloSamples& samples,
   IntegerCaloSamples output(samples.id(), tpSamples);
   output.setPresamples(tpPresamples);
 
+  // Based on the |ieta| of the sample, retrieve the correct region "coded" veto threshold
+  // where two of the possible values have special meaning
+  unsigned int codedVetoThreshold = codedVetoThresholds_[theIeta];
+
+  // Anything in range (1, 2048) inclusive shall activate the veto
+  unsigned int actualVetoThreshold = codedVetoThreshold;
+  bool applyVetoThreshold = codedVetoThreshold > 0 && codedVetoThreshold <= 2048;
+
+  // Special value to disable vetoing in the PFA1' algo is 0
+  if (codedVetoThreshold > 0 && codedVetoThreshold <= 2048) {
```

Comment:
I am noticing that you are here changing the logic with respect to the previous version: previously, the LogWarning was issued only for `codedVetoThreshold > 2048`, while now it is also issued for zero (in particular!) and negative values of it.
This is floading the logs of warnings, and I don't think that this is wanted...

I'd suggest to apply the suggestion in https://github.com/cms-sw/cmssw/pull/44373#discussion_r1536585668

Reply:
Ah, yep, I see, applied with your suggestion and squashed the commits down.